packages/components/organisms/f-content-cards/package.json

```
@@ -51,17 +51,19 @@
   },
   "peerDependencies": {
     "@justeat/browserslist-config-fozzie": ">=1.1.1",
-    "vue": "2.x"
+    "vue": "2.x",
+    "@braze/web-sdk": "3.3.0"
```

Let's make the `@braze/web-sdk` `peerDependencies` requirement a bit more permissive, to allow for minor version bumps:
```suggestion
    "@braze/web-sdk": "^3.3.0"
```

---

changed to `>=` as per the braze adapter way of doing it